07-Modelling-Patchy-Particles/02-Simulating-a-System-of-Patchy-Particles-with-HPMC.ipynb

```diff
@@ -66,7 +66,7 @@
    "metadata": {},
```

Comment:
<p>Replace</p><blockquote>As discussed in the Logging tutorial,</blockquote><p>with </p><blockquote>As discussed in the [Logging tutorial](../02-Logging/02-Saving-Array-Quantities.html),</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/131/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>